server/chat-commands.js

```diff
@@ -3748,6 +3748,26 @@ const commands = {
 		this.parse(`/help importinputlog`);
 	},
 
+	acceptdraw: 'offerdraw',
+	offerdraw(target, room, user) {
+		const battle = room.battle;
+		if (!battle) return this.errorReply("This command can only be used in battle rooms.");
+		if (!this.can('roomvoice', null, room)) return;
```

Comment:
Should I just make it so players can use it? If a room owner or above wants to draw said room owner can just use `/forcetie`.

Reply:
Yes, players should be able to use it. Is that not the case?

Reply:
> only players

Reply:
Eh, I guess it's useful sometimes, supported it.

Reply:
It doesn't make sense for ROs/leaders to offer a draw – you'd still need both players to agree to it, right?

Reply:
Yeah, I know the code does that. Leader offers a deal, if both players accept the battle is tied.

Reply:
Oh, that's not a bad idea.